tests/acceptance/infinity-route-with-meta-test.js

```diff
@@ -32,11 +32,11 @@ test('it works with parameters', function(assert) {
   andThen(() => {
     assertDetails(assert, {
       title: "Listing Posts using Parameters",
-      listLength: 2,
-      reachedInfinity: false
+      listLength: 3,
+      reachedInfinity: true
```

Comment:
So this test stumped me.  Even though `per_page=2`, another request is still made, so I changed it to 3.  I'm thinking it was due to CSS height/transform on the ember-testing container.

Reply:
hmmm.  It's not related to the debounce issue I mentioned above?

Reply:
I believe it is because of the viewport size issue...caused by CSS changes in ember-qunit.  So before, this test should have fetched the last page `reachedInfinity: false`.  But b/c of the CSS changes, 3 now fit in the viewport.

Reply:
Hmm ok!  That's likely it.  I wonder if there's a way to hard code the qunit driver viewport dimensions